dummy

```diff
@@ -0,0 +1 @@
+tummy test
```

Comment:
Got this too :)

Michelle 

Sent from my iPhone

> On Apr 8, 2016, at 8:07 PM, mimiyan <notifications@github.com> wrote:
> 
> In dummy:
> 
> > @@ -0,0 +1 @@
> > +tummy test
> @michelleli94 It is just a test to see if you can receive an email notification when I mention you at PRs.
> 
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
>